service/service.go

```diff
@@ -34,6 +34,7 @@ var (
 
 type ExtProcessor struct {
 	filters []filter.Filter
+	streams []filter.Stream
```

Comment:
Feel like naming wise this is quite confusing, especially because we have many streams in the context of a stream.

Reply:
streamCallbacks?

Reply:
I guess it's not in the public API anyway, so let's do that